src/util.h

```diff
@@ -67,6 +67,14 @@ extern "C" {
 
 #define COMPILE_ERROR_ON(cond) ((void)sizeof(char[(cond) ? -1 : 1]))
 
+#define pf_always_inline __attribute__((always_inline)) inline
```

Comment:
One other way I use sometimes, is:

#ifdef HAS_GNU_ATTR
#define attribute __attribute__
#define attribute(...) __attribute__((__VA_ARGS__))
#else
#define attribute(...)
#endif

Or just define this macro in CFLAGS